src/modules/constants.ts

```diff
@@ -160,6 +161,51 @@ const POLYGON_NETWORK: NetworkInfo = {
   gnosisTxService: 'https://safe-transaction.polygon.gnosis.io/',
 };
 
+const RINKEBY_TEST_NETWORK: NetworkInfo = {
+  name: 'Rinkeby Testnet',
+  chainId: 4,
+  shortName: 'RIN',
+  contractAddressLink: '',
+  gnosisTxService: 'https://safe-transaction.rinkeby.gnosis.io/',
+};
+
+/*
+ * @NOTE: Although officially supported, 
```

Comment:
While useful for testing, I was trying to avoid having testnets available as options in the Chains. For ones that don't have a transaction service, we can't use anyway. So, we can exclude those.

Reply:
Ok thanks, I will update to recognise this.